README.md

```diff
@@ -1,4 +1,4 @@
-![Cellotape mascot](./cellotape-gopher.png)
+![Cellotape mascot](./Cellotape-Gopher.png)
```

Comment:
The file name is still `cellotape-gopher.png` in this PR.
Notice that case sensitive only file name changes are ignored by regular `git add .`.
You can use this command to tell git the file is reanmed:
`git mv -f cellotape-gopher.png Cellotape-Gopher.png`

Reply:
Sorry, overzealous correction of capitalization on "go". Reverting this.